### ISSUE TITLE
torrenttools edit (--announce) crashing on Windows

### ISSUE THREAD

**Issue by RibShark** [fbdtemme/torrenttools#24]
On the Windows version, I am using the command `torrenttools edit -a "http://newtracker.url/announce.php" "A.Release.by.a-GROUP.torrent" `, which crashes, producing no output. Afterwards, using `torrenttools info "A.Release.by.a-GROUP.torrent"` shows that the announce URL has not been edited. I have also tried using the `-o` option, as well as using a named tracker (which doesn't update any of the relevant parts of the torrent).

On Linux, this command works as intended.

Event Viewer log: 
````
Faulting application name: torrenttools.exe, version: 0.0.0.0, time stamp: 0x610e32a1
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00007ff7ab3c0000
Faulting process ID: 0x3680
Faulting application start time: 0x01d797908bc46f20
Faulting application path: C:\Program Files\torrenttools\torrenttools.exe
Faulting module path: unknown
Report ID: 541fa805-e161-4d41-8cea-92aeb9bce5e1
Faulting package full name: 
Faulting package-relative application ID: 
````

**Comment by fbdtemme** [comment]
The segfaults seem to originate in the re2 regex engine and I think are the result of interactions with some mingw issues with thread_local storage specifier. I have opened an issue in the re2 repository: https://github.com/google/re2/issues/325.
For now I was able to fix this by reverting to an older version.

Thanks for reporting and sorry for my slow response.

**Comment by fbdtemme** [comment]
Fixed with release 0.6.2
